Comparison Engine/Comparison Engine/Child Forms/UserContribution.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Comparison_Engine.Base_Classes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace Comparison_Engine.Child_Forms
+
+{
+    public partial class UserContribution : Form
+    {
+        public UserContribution(Drink drink, Bar bar)
+        {
+            InitializeComponent();
+        }
+
+        
+        private void buttonAddDrink_Click(object sender, System.EventArgs e)
+        {
+        }
+        private void buttonAddBar_Click(object sender, System.EventArgs e)
+        {
+            string BarName = textBoxBarName.Text;
+            string BarAdress = textBoxBarAdress.Text;
+            //AddBar(BarName, BarAdress);
+
+        }
```

Comment:
Should use BarManager class to add bars and drinks. Also some ID generation should probably appear somewhere in this class

Reply:
ID's, for now, are generated when you create a bar using the BarManager.AddBar method